TAO/tao/Storable_Base.h

```diff
@@ -112,6 +112,33 @@ namespace TAO
     virtual Storable_Base& operator << (ACE_INT64 ) = 0;
     virtual Storable_Base& operator >> (ACE_INT64 &) = 0;
 
+#if defined (ACE_HAS_CPP11)
```

Comment:
Are these changes still required now that there are no changes anymore in ace/Basic_Types.h?

Reply:
Yes, some code, e.g., `read` and `write` methods of the `orbsvcs/orbsvcs/PortableGroup/PG_Group_List_Store`class need these overloaded functions because some operands are `unsigned long` but there is no existing `operator<<` for `unsigned long`. Similar to `be_helper.h`, this may change after we have other C++11 types added.